internal/pkg/utils/utils.go

```diff
@@ -94,6 +95,25 @@ func nodenameFromFile(filename string) string {
 	return string(data)
 }
 
+// MTUFromFile reads the /var/lib/calico/mtu file if it exists and
+// returns the nodename within.
+func MTUFromFile(filename string) (int, error) {
+	if filename == "" {
+		filename = "/var/lib/calico/mtu"
+	}
+	data, err := ioutil.ReadFile(filename)
+	if err != nil {
+		if os.IsNotExist(err) {
+			// File doesn't exist, return zero.
+			logrus.Infof("File %s does not exist", filename)
+			return 0, nil
+		}
+		logrus.WithError(err).Errorf("Failed to read %s", filename)
+		return 0, err
+	}
+	return strconv.Atoi(string(data))
```

Comment:
Recommend slipping in a `strings.TrimSpace()` if `Atoi` doesn't ignore spaces just to make the mtu file harder to manually break